src/pages/PeoplePage.tsx

```diff
@@ -0,0 +1,122 @@
+import { useEffect, useMemo, useState } from 'react';
+
+import { Loader } from '../components/Loader';
+import { PeopleTable } from '../components/PeopleTable';
+import { getPeople } from '../services/api';
+import { Person } from '../types';
+import { PeopleFilters } from '../components/PeopleFilters';
+import { useSearchParams } from 'react-router-dom';
+import { SortFilter } from '../types/SortFilter';
+import { Order } from '../types/OrderFilter';
+
+export const PeoplePage = () => {
+  const [people, setPeople] = useState<Person[]>([]);
+  const [isLoading, setIsLoading] = useState(true);
+  const [isError, setIsError] = useState(false);
+  const [searchParams] = useSearchParams();
+
+  const query = searchParams.get('query')?.toLowerCase().trim();
+  const sex = searchParams.get('sex');
+  const centuries = searchParams.getAll('centuries');
+  const sort = searchParams.get('sort');
+  const order = searchParams.get('order');
+
+  useEffect(() => {
+    getPeople()
+      .then(data => {
```

Comment:
It's better to use async/await syntax with try/catch block for handling promises and errors. It's more readable and less error-prone.